app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,10 @@
     <color name="color_black">#000</color>
```

Comment:
7 -> 15 color name em nên đặt tên đúng hơn.
Có thể paste vào đây để xem tên
https://www.color-blindness.com/color-name-hue/

Reply:
http://chir.ag/projects/name-that-color/#6195ED